scripts/cf-operator-wait.sh

```diff
@@ -19,13 +19,11 @@ for crd in "${crds[@]}" ; do
 done
 
 # Wait for the operator deployments to be ready
-deployments=(
-    cf-operator-quarks
-    cf-operator-quarks-job
-    cf-operator-quarks-secret
-    cf-operator-quarks-statefulset
-)
-for deployment in "${deployments[@]}" ; do
+deployments=$(kubectl get deployments --namespace cf-operator \
+                       --no-headers -o custom-columns=":metadata.name" \
+                  | tr '\r\n' ' ');
```

Comment:
```suggestion
mapfile -t deployments < <(kubectl get deployments --namespace cf-operator --output=name)
```

And then drop the `deployment/` prefix when we use it (since that's included in `--output=name`).

Reply:
Fixed in https://github.com/cloudfoundry-incubator/kubecf/pull/1611/commits/d47783193a7553735981823540830fb0aa61d8f8
Committed it myself, sorry (I rebased master at the beginning, and later did a merge from master instead, cherry-picking my commit for this..).